src/pages/buy-sell/screens/BuySellTable/BuySellTable.tsx

```diff
@@ -18,11 +20,25 @@ const BuySellTable = () => {
     const { queryString, setQueryString } = useQueryString();
     const activeTab = queryString.tab || ADVERT_TYPE.BUY;
 
-    const [selectedCurrency, setSelectedCurrency] = useState<string>(p2pSettingsData?.localCurrency || '');
-    const [sortDropdownValue, setSortDropdownValue] = useState<TSortByValues>('rate');
+    const {
+        filteredCurrency,
+        selectedPaymentMethods,
+        setFilteredCurrency,
+        setSortByValue,
+        shouldUseClientLimits,
+        sortByValue,
+    } = useBuySellFiltersStore(
+        useShallow(state => ({
```

Comment:
```suggestion
        useShallow(({ filteredCurrency, selectedPaymentMethods, setFilteredCurrency, setSortByValue, shouldUseClientLimits, sortByValue })=> ({
```

Reply:
IMO its better we follow what i had done here, cause it'll just add even more lines to the code like this:

![Screenshot 2024-07-02 at 10 36 08 AM](https://github.com/deriv-com/p2p/assets/103412909/35a70aad-0458-4db5-a522-77c7eaa18c33)

Reply:
i think in this case, its not necessary to destructure the state